### ISSUE TITLE
Unable to navigate command palette with keyboard

### ISSUE THREAD

**Issue by Shunseii** [abuibrahim2/quranlookup#34]
السلام عليكم ورحوة الله وبركاته يا أخي الكريم

جراك الله خيرا ويحفظك على إنشاء هذا التطبيق

I noticed that keyboard navigation using the arrow keys doesn't work in the command palette with this plugin.

I think something is interfering with it in the Qur'an search modal code.

The workaround for now is just disabling this plugin when not using it, but it's not ideal.

بارك الله فيك

**Comment by AE-SAY-WAY** [comment]
I had the same issue

**Comment by muhammadammarzahid** [comment]
Salam alaikum. Facing the same issue.

**Comment by Hasko7** [comment]
Salam Alaikum, i also noticed the same issue.
